apps/frontend/core-dapp/src/features/deposit/DepositWarning.tsx

```diff
@@ -0,0 +1,108 @@
+import { observer } from 'mobx-react-lite'
+import styled from 'styled-components'
+import { Alert as BaseAlert, Icon, spacingIncrement } from 'prepo-ui'
+import { useRootStore } from '../../context/RootStoreProvider'
+import { compactNumber, formatUsd } from '../../utils/number-utils'
+
+const Alert = styled(BaseAlert).attrs({
+  color: 'neutral1',
+  icon: (
+    <Icon name="info" color="warning" width={spacingIncrement(20)} height={spacingIncrement(20)} />
+  ),
+  showIcon: true,
+  type: 'warning',
+})`
+  &&& {
+    .ant-alert-message {
+      font-size: ${({ theme }): string => theme.fontSize.xs};
+      font-weight: ${({ theme }): number => theme.fontWeight.medium};
```

Comment:
Let's give it 16px line height

```suggestion
    .ant-alert-message {
      line-height: ${spacingIncrement(16)};
      font-size: ${({ theme }): string => theme.fontSize.xs};
      font-weight: ${({ theme }): number => theme.fontWeight.medium};
```